src/main/java/com/hyperlocal/server/Utilities.java

```diff
@@ -17,4 +17,13 @@ public static boolean doubleThresholdCompare(Double numA, Double numB, Double th
     {
         return Math.abs(numA-numB) < threshold;
     }
+
+    // Create a ?,?,? placeholder with numOfPlaceholders '?' to use in SQL prepared statements
+    public static String getPlaceHolderString(Integer numOfPlaceholders) {
+      StringBuilder result = new StringBuilder();
+      for (Integer i = 0; i < numOfPlaceholders; i++) {
+        result.append("?,");
+      }
+      return result.length() > 0 ? result.substring(0, result.length() - 1) : "";
```

Comment:
I think it's fair to assume `numOfPlaceholders >= 1`. In that case:
```suggestion
      StringBuilder result = new StringBuilder("?");
      for (Integer i = 0; i < numOfPlaceholders-1; i++) {
        result.append(",?");
      }
      return result.toString();
```